README.md

```diff
@@ -4,45 +4,68 @@ for emitting entities for the [Kythe](https://kythe.io) indexing schema.
 
 This is not an official Google product.
 
-# Instructions
+# Installation (Linux)
 
-## Prerequisites
+## Install Haskell compiler & tools (GHC & Stack).
 
-### Protoc 3
+See: https://www.haskell.org/downloads#platform
 
-First, install the `protoc` binary somewhere in your PATH. You can get it by
-downloading the corresponding file for your system from
-https://github.com/google/protobuf/releases. The corresponding file will be
-named something like `protoc-*-.zip`.
+If you only miss Stack, get it from here: http://docs.haskellstack.org/en/stable/README/
 
-Note: you'll need at least version 3 of the protobuf compiler. Check the version
-if you intend to install the one provided by your package manager.
+## Kythe
 
-### Kythe
+If you want to use the Kythe frontend, download a [Kythe
+release](https://github.com/google/kythe/releases) and unpack it.
 
-If you want to use the Kythe frontend, it is advised to have the Kythe tools
-installed.
+```
+tar xzf kythe-v*.tar.gz
+rm -rf /opt/kythe
+mv kythe-v*/ /opt/kythe
+```
 
-Please install a [Kythe release](https://github.com/google/kythe/releases) to
-`/opt/kythe` for this. Version `v0.0.26` was tested.
+Version `v0.0.26` is verified to work with Haskell indexer.
 
-### Docker
+## Protoc 3
 
-If you plan to use the Dockerized build feature of stack, please install
-Docker. It is also advised to set up a docker wrapper script by following the
-instructions at the [stack Docker
-security](https://docs.haskellstack.org/en/stable/docker_integration/#security)
-section.
+Download the latest [Proto compiler 3
+release](https://github.com/google/protobuf/releases), unpack it and place the
+binary in the PATH.
+
+```
+unzip -j protoc-*-linux-x86_64.zip bin/protoc -d /usr/local/bin/
+```
 
-## Build
+# Build the project
 
 Use the following to build and run tests:
 
-    git clone --recursive https://github.com/google/haskell-indexer.git
-    cd haskell-indexer
-    stack build
+```
+git clone --recursive https://github.com/google/haskell-indexer.git
```

Comment:
Note: it might be needed to init subrepos (see commands at beginning of one of the scripts). Or does `--recursive` init them automatically? Could add a comment.